doc/source/relationships.rst

```diff
@@ -23,7 +23,7 @@ Remember this needs to be declared on both sides of the definition::
         car = RelationshipTo('Car', 'CAR', cardinality=One)
 
     class Car(StructuredNode):
-        owner = RelationshipFrom('Person', cardinality=One)
+        owner = RelationshipFrom('Person', 'CAR', cardinality=One)
```

Comment:
wouldn't this rather be `'OWNED_BY` or similar?

Reply:
afaik, that argument sets the [relationship type](https://neo4j.com/docs/developer-manual/current/terminology/#term-relationship-type). the relationship source is set when the model class is initialized.

Reply:
@funkyfuture yes you're right. Let me fix that.

Reply:
As mentioned, in the comments, I had this right originally. The relationship name should be `CAR` because we want to access the `CAR` relationship from the `Person` node.